doc/book/src/config.md

```diff
@@ -104,8 +99,13 @@ Maelstrom programs also support the `--config-file` (`-c`) command-line option.
 If this option is provided, the specified configuration file, and only that
 file, will be used.
 
+v---- Interesting. I suppose I would have assumed `-` would mean "read config from stdin".
+
 If `--config-file` is given `-` as an argument, then no configuration file is used.
```

Comment:
Yeah, that's definitely the default meaning for -. But you can't really read the config file from stdin, so it seemed like a decent overload of it. I'm not sure it's even useful.